docker/README.md

```diff
@@ -0,0 +1,105 @@
+# [Devito] docker image library
```

Comment:
Drop the square brackets (otherwise u get tons of useless links, see [here](https://github.com/devitocodes/devito/tree/docker-with-bases/docker#devito-images)

Reply:
prod

Reply:
I like the links, they don't cost anything

Reply:
fine no strong objections but they make the page look like a rainbow lol